modules/CmpdReg/src/client/src/SearchForm.js

```diff
@@ -7,7 +7,7 @@ $(function() {
             _.each(attributes, function(att, key) {
                 if( (att != '' && att!=null) && key!='aliasContSelect' && key!='searchType' && key!='percentSimilarity' && key!='maxResults' && key!='loggedInUser' ) {
                     if( key=='chemist') {
-                        if( att.get('code')!='anyone') {
```

Comment:
Is there a corresponding place where this "anyone" chemist is defined that needs to be updated to go with the new scheme of chemist-as-string instead of chemist-as-object?